lib/exq/manager/server.ex

```diff
@@ -1,22 +1,17 @@
 defmodule Exq.Manager.Server do
   require Logger
   use GenServer
-  alias Exq.Stats.Server, as: Stats
   alias Exq.Enqueuer
-  alias Exq.Redis.JobQueue
   alias Exq.Support.Config
-
-  @default_name :exq
```

Comment:
This changes the default semantics of how to use Exq, correct?  I'm assuming we will have to do this:
```elixir
Exq.enqueue(Exq, "default", "BusyWorker", [])
```
instead of 
```elixir
Exq.enqueue(:exq, "default", "BusyWorker", [])
```
I think that looks good, maybe we can update the docs?   

I can also see there being a method that just defaults to the correct name as well, so a user can skip the first arg if not needed.